definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -66,5 +66,95 @@
     "FieldType": "FixedList",
     "FieldTypeParameter": "ServiceApplicationPaymentTypeEnum",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "PBANumber",
+    "Label": "Enter your account number",
+    "HintText": "Example: PBA0896124",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "PBAReference",
+    "Label": "Enter your reference",
+    "HintText": "This will appear on your statement to help identify your payment",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "HWFReference",
+    "Label": "Enter your HWF reference",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "05/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "generalApplicationWithoutNoticeFeeSummary",
+    "Label": "Here are your payment details",
+    "FieldType": "OrderSummary",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "12/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServiceApplicationPaymentFeeAccountLabel",
+    "Label": "Payment Method: Fee Account",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "12/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServiceApplicationPaymentHWFLabel",
+    "Label": "Payment Method: Help with fees",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "12/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServiceApplicationPaymentTelephoneLabel",
+    "Label": "Payment Method: Telephone",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "12/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServiceApplicationPaymentChequeLabel",
+    "Label": "Payment Method: Cheque",
```

Comment:
Why didn't you use variables (like the examples below this line)?

Reply:
Yeah this is something i dont personally like. 
When I used variables, i got answers like "feeAcount" and "helpWithFees". It seemed to take the ID and not the name. A workaround was to just hard code the specific names under conditions. 
I dont like it but it seemed to be a way to get around CCDs limitations